Dangerfile

```diff
@@ -1,13 +1,50 @@
 # frozen_string_literal: true
 
+def wip_feature?
+  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
+  has_wip_title = github.pr_title.include?('WIP')
+
+  has_wip_label || has_wip_title
+end
+
+# these are reviewers actively providing feedback and potentially changing the state of the PR (approved, changes-requested)
+def active_reviewers?
+  repo_name = github.pr_json['base']['repo']['full_name']
+  pr_number = github.pr_json['number']
+
+  !github.api.pull_request_reviews(repo_name, pr_number).empty?
+end
+
+# requested_teams / requested_reviewers are users initially requested to review a PR, who haven't reacted yet
+def requested_reviewers?
+  has_requested_reviews = !github.pr_json['requested_teams'].to_a.empty? || !github.pr_json['requested_reviewers'].to_a.empty?
+  has_requested_reviews || active_reviewers?
+end
+
+return if github.pr_labels.include?('Releases')
+
 github.dismiss_out_of_range_messages
 
-# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
-warn('PR is classed as Work in Progress') if github.pr_title.include? '[WIP]'
+manifest_pr_checker.check_all_manifest_lock_updated
 
-rubocop.lint inline_comment: true, fail_on_inline_comment: true
+labels_checker.check(
```

Comment:
@iangmaia how can I change that to be a warning instead of a build failure?

Reply:
So now it requires at least one label by using the RegEx `//` and the `required_` parameters.

You can change it to warnings by using the `recommended_` parameters (see the [source](https://github.com/Automattic/dangermattic/blob/9ba0e5824e66e16526897a4583c9b900f9b8635c/lib/dangermattic/plugins/labels_checker.rb#L43C10-L43C10) of the check).

So an equivalent code but showing a warning instead of an error should be something like:
```ruby
labels_checker.check(
  recommended_labels: [//],
  recommended_labels_warning: 'PR requires at least one label.'
)
```

Reply:
Thank you!